index.html

```diff
@@ -10,54 +10,62 @@
   <title>My Portfolio</title>
 </head>
 <body>
+  <div class="header-bg">
+    <header>
+      <nav class="navbar">
+        <a href="">Portfolio</a>
+        <a href="">About</a>
+        <a href="">Contact</a>
+      </nav>
+      <div id="union">
+        <img src="images/Icons/Icon - Menu.png" alt="union">
+      </div>
+    </header>
 
-  <header>
-    <nav id="union">
-      <img src="images/Icons/Icon - Menu.png" alt="union">
-    </nav>
-
-  </header>
+    <section class="headline">
 
-  <section class="headline">
+      <div class="headline-text">
+        <h1>
+          Hey there.
+          <br>
+          I’m Didy.
+          <br>
+          <span class="green">I’m a Full Stack developer from Microverse</span>
+        </h1>
 
-    <div class="headline-text">
-      <h1>
-        Hey there.
-        <br>
-        I’m Didy.
-        <br>
-        <span class="green">I’m a Full Stack developer from Microverse</span>
-      </h1>
+        <p class="spacing-default">
+          I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+          don’t hesitate to contact me.
+        </p>
+
+        <ul class="social spacing-default">
+          <li><a href="https://twitter.com/diarisdiakite">
+            <i class="fab fa-twitter"></i>
+          </a></li>
+          <li><a href="https://www.linkedin.com/diarisdiakite/">
+            <i class="fab fa-linkedin-in"></i></li>
+          </a></li>
+          <li><a href="https://www.medium.com/#">
+            <i class="fab fa-medium"></i>
+          </a></li>
+          <li><a href="https://www.github.com/diarisdiakite">
+            <i class="fab fa-github"></i>
+          </a></li>
+          <li><a href="https://www.angellist.com/#">
+            <i class="fab fa-angellist"></i>
+          </a></li>
+        </ul>
+      </div>
```

Comment:
- [ ] Please add some gap between the social icons on the top section to make your design look identical to the [figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=0dp0MALuxE1qoyAu-0)

figma design

<img width="347" alt="Screenshot 2023-03-31 at 9 27 09 AM" src="https://user-images.githubusercontent.com/44798044/229030303-5d51cdf6-70bd-43ab-8094-f25bee40f8c7.png">

your design:

<img width="708" alt="Screenshot 2023-03-31 at 9 27 16 AM" src="https://user-images.githubusercontent.com/44798044/229030281-c2cd2e2b-af81-4a64-9a15-d8e83e67a966.png">